src/Service/QuizTemplateService.php

```diff
@@ -46,15 +46,16 @@ public function getQuiz(int $id): QuizTemplate
         return $quiz;
     }
 
```

Comment:
I would rename this method to be more transparent with what it does. getFilteredQuizez or getQuizezByFilter, getQuizezFilterdQuizezForPage etc. getQuizzes leads me to believe all quizez are returned